Pokemon.java

```diff
@@ -0,0 +1,135 @@
+import java.util.Arrays;
+import java.util.List;
+
+abstract class Pokemon {
+    private String name;
+    private int level;
+    private int hp;
+    private final int maxHP;
+    private int xp;
+    private final int defence;
+    private final int attack;
+    private String food;
+    private final int accuracy;
+    private String[] pokemonType;
+
+    public Pokemon(String name, int level, String[] pokemonType) {
+        this.name = name;
+        this.level = level;
+        this.maxHP = level * 10;
+        this.hp = maxHP;
```

Comment:
Wat goed dat je deze waarden zo berekent. Je hoeft inderdaad niet alle instance variabelen via de constructor parameters te vullen. Dat maakt ook dat je geen elle lange lijst van argumenten hoeft in te vullen in Main, wanneer je een niewe instance aanmaakt.